pages/regions/cancel_deployment_modal.py

```diff
@@ -0,0 +1,43 @@
+#
+# $Id$
+#
+from selenium.webdriver.common.by import By
+
+from base import Base
+
+
+class CancelDeploymentModalPage(Base):
+    """Cancel Deployment Modal Window:
+    1) Modal Header
+    2) Close button
+    3) Modal Body
+    4) Modal Footer
+    5) Exit and Delete button
+    6) Exit and Save button
+    7) Continue Working button"""
+
+    _page_title = "QuickStart Cloud Installer"
+
+    _exit_and_delete_button_locator = (
+        By.XPATH, "//button[contains( text(), 'Exit and Delete' )")
+
+    @property
+    def exit_and_delete(self):
+        return self.selenium.find_element(*self._exit_and_delete_button_locator)
+
+    _exit_and_save_button_locator = (
```

Comment:
Most of the other selenium classes puts all the locators up at the top of the class.   I can see arguments for either approach...I just think we should be consistent.